tests/unit/options.js

```diff
@@ -89,7 +89,7 @@
           selectedMax: 2
       });
 
-      checkboxes = el.multiselect("widget").find(":checkbox");
+      var checkboxes = el.multiselect("widget").find(":checkbox");
```

Comment:
Good catch adding all these vars.

Reply:
Actually, this is a result of you updating Qunit.  👍 

It has a new option you can set when running it to find all the global variables and flag them as test failures.